server/models/Department.js

```diff
@@ -1,33 +1,29 @@
 const { Model, DataTypes } = require('sequelize');
 const sequelize = require('../config/connection');
 
-class Department extends Model { }
+class Department extends Model {}
 
 Department.init(
-    {
-        id: {
-            type: DataTypes.INTEGER,
-            primaryKey: true,
-            autoIncrement: true,
-        },
-        name: {
-            type: DataTypes.STRING,
-        },
-        company_id: {
-            type: DataTypes.NUMBER,
-            references: {
-                model: 'company',
-                key: 'id',
-            },
-        },
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
     },
-    {
-        sequelize,
-        timestamps: false,
-        freezeTableName: true,
-        underscored: true,
-        modelName: 'department',
-    }
+    name: {
+      type: DataTypes.STRING,
+    },
+    company_id: {
```

Comment:
Remove the entire `company_id` column. The relationship creates it automatically